GoogleTestNuGet/googletest.targets.tt

```diff
@@ -16,28 +16,28 @@
   </ItemGroup>
   <ItemDefinitionGroup Condition="'$(Disable-<#= PackageNameDashes #>)' == ''">
     <Link>
-      <AdditionalDependencies Condition="'$(Configuration)' == 'Debug' And '$(Platform)' == 'x64'">$(MSBuildThisFileDirectory)..\..\\<#= PathToBinaries #>\x64\Debug\gtestd.lib;%(AdditionalDependencies)</AdditionalDependencies>
```

Comment:
There are runtimes errors with this patch due to mismatches detected by the linker.
![image](https://github.com/microsoft/TestAdapterForGoogleTest/assets/100718117/288b2988-2e08-4b59-8e29-afb13963dd0d)

Reply:
Thank you for testing